main.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// A Recipe Struct allows you to insert recipe documents into your
+// collection
+
+type Recipe struct {
+	Name              string
+	Ingredients       []string
+	PrepTimeInMinutes int `json:"prepTimeInMinutes" bson:"prepTimeInMinutes"`
+}
+
+func main() {
+
+	// TODO:
+	// Replace the placeholder connection string below with your
+	// Altas cluster specifics. Be sure it includes
+	// a valid username and password! Note that in a production environment,
+	// you do not want to store your password in plain-text here.
+	var mongoUri = "<Your Atlas Connection String>"
+
+	// CONNECT TO YOUR ATLAS CLUSTER:
+	ctx, _ := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
As a best practice, let's make sure to defer canceling the context, e.g.:
```
ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
defer cancel()
```

Reply:
👍